src/webgl/p5.Framebuffer.js

```diff
@@ -931,6 +950,238 @@ class Framebuffer {
     callback();
     this.end();
   }
+
+  loadPixels() {
+    const gl = this.gl;
+    const prevFramebuffer = gl.getParameter(gl.FRAMEBUFFER_BINDING);
+    gl.bindFramebuffer(gl.FRAMEBUFFER, this.framebuffer);
+    const colorFormat = this._glColorFormat();
+    this.pixels = readPixelsWebGL(
+      this.pixels,
+      gl,
+      this.framebuffer,
+      0,
+      0,
+      this.width * this.density,
+      this.height * this.density,
+      colorFormat.format,
+      colorFormat.type
+    );
+    gl.bindFramebuffer(gl.FRAMEBUFFER, prevFramebuffer);
+  }
+
+  /**
+   * Get a region of pixels, or a single pixel, from the canvas.
+   *
+   * Returns an array of [R,G,B,A] values for any pixel or grabs a section of
+   * an image. If the Framebuffer has been set up to not store alpha values, then
+   * only [R,G,B] will be returned. If no parameters are specified, the entire
+   * image is returned.
+   * Use the x and y parameters to get the value of one pixel. Get a section of
+   * the display window by specifying additional w and h parameters. When
+   * getting an image, the x and y parameters define the coordinates for the
+   * upper-left corner of the image, regardless of the current <a href="#/p5/imageMode">imageMode()</a>.
+   *
+   * @method get
+   * @param  {Number}         x x-coordinate of the pixel
+   * @param  {Number}         y y-coordinate of the pixel
+   * @param  {Number}         w width
+   * @param  {Number}         h height
+   * @return {p5.Image}       the rectangle <a href="#/p5.Image">p5.Image</a>
+   */
+  /**
+   * @method get
+   * @return {p5.Image}      the whole <a href="#/p5.Image">p5.Image</a>
```

Comment:
Yep! This is the behaviour for p5.Graphics too. Before framebuffers, using `.get()` to turn a graphic into a `p5.Image` is a pattern I'd use to get slightly better performance when using a static graphic as a WebGL texture to avoid it calling `gl.texImage2D` each frame to send the pixel data to the GPU.